agent/php_nrini.c

```diff
@@ -1303,6 +1303,40 @@ static PHP_INI_MH(nr_tt_max_segments_web_mh) {
   return SUCCESS;
 }
 
+static PHP_INI_MH(nr_span_events_max_samples_stored_mh) {
+  nriniuint_t* p;
+  int val;
+
+#ifndef ZTS
+  char* base = (char*)mh_arg2;
+#else
+  char* base = (char*)ts_resource(*((int*)mh_arg2));
+#endif
+
+  p = (nriniuint_t*)(base + (size_t)mh_arg1);
+
+  (void)entry;
+  (void)mh_arg3;
+  NR_UNUSED_TSRMLS;
+
+  /* Anything other than a valid value will result in the
+   * default value of NR_DEFAULT_SPAN_EVENTS_MAX_SAMPLES_STORED.
+   */
+
+  p->where = 0;
+  val = (int)strtol(NEW_VALUE, 0, 0);
+
+  if (0 != NEW_VALUE_LEN) {
+    if ((0 >= tempVal) || (NR_MAX_SPAN_EVENTS_MAX_SAMPLES_STORED < tempVal)) {
+      val = NR_DEFAULT_SPAN_EVENTS_MAX_SAMPLES_STORED;
```

Comment:
For the ini, the current convention is that none of the other values issue a message when they override invalid values with the default.  I think this is a value a customer could easily check with phpinfo.
That being said, it could easily be sent to our logfile, but it's best to keep things consistent.
What are the thoughts about changing the convention for all or keeping the current convention?

Reply:
Might be a handy support tool to be able to quickly look in the log and see if any problematic values existed in the ini file.

Reply:
> 'be able to quickly look in the log and see if any problematic values existed in the ini file'

Was my argument behind my [comment](https://github.com/newrelic/newrelic-php-agent/pull/388#discussion_r836473201).